lib/Parser/Parser.cpp

```diff
@@ -576,10 +592,19 @@ bool Parser::typeCheckInst(Inst::Kind IK, unsigned &Width,
   // overflow instruction respectively. The second element of
   // ExtractValue instruction is an index value. We don't type check
   // the operands width as the two elements vary in width.
-  if (IK != Inst::ExtractValue)
+  if (IK != Inst::ExtractValue) {
     if (!typeCheckOpsMatchingWidths(OpsMatchingWidths, ErrStr))
       return false;
 
+    for (auto Op : Ops) {
+      if (isOverflow(Op->K)) {
+        ErrStr = std::string("overflow intrinsic cannot be an operand of ") + Inst::getKindName(IK) +
```

Comment:
I don't think both of these explicit casts to std::string are needed

Reply:
It does give build error if I don't typecast explicitly. For the other one, which doesn't combine multiple strings, I don't need typecast. I have fixed that.